### DISCUSSION TITLE
Custom Fonts Installation on Multisite WordPress Network

### DISCUSSION THREAD

**Discussion by anadelgado96** [elementor/elementor#23286] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I'm the manager of a massive multisite WordPress network with over 100 websites, and we're facing a font-related challenge. We want to install custom fonts across all the subsites, but doing it manually for each one is turning into a real pain, since each font has to be uploaded from the Elementor > Custom fonts section.

### Describe the solution you'd like

It would be great if Elementor had an option to share Global custom fonts on all subsites that are under a multisite network. We have other similar issues with not being able to share Global templates across all the network and update them on the main site but that's a separate issue, for now it would be great if at least we could access custom fonts from the main site on each subsite without having to install them individually.

### Describe alternatives you've considered

I have considered uploading the fonts to the header of the 'Plain Text' theme, which is installed on all subsites, however, this solution would not automatically load the new custom fonts on the Elementor editor, so if we wanted to use the new fonts we would need to apply custom CSS for all the headings and content, which could imply the same effort as uploading the fonts  through Elementor > Custom fonts  on each subsite.

### Additional context

We would really appreciate if there was a way of sharing global fonts/templates across a multisite network.

**Comment by Bovelett** [comment]
Great Feature Request, @anadelgado96. I have customers juggling massive multi-sites as well. It would be a fantastic option to have.
